ansible/all-in-one/roles/sockets/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+- copy:
+    src: "{{ item }}"
+    dest: /etc/systemd/system/
+  loop:
+    - http.socket
+    - https.socket
+    - http.service
+    - https.service
+  notify:
+    - Reload systemd
+
+- name: Reload systemd
+  meta: flush_handlers
```

Comment:
What's that?

Reply:
I mean, I don't understand what it does. What is `flush_handlers`? How it's related to systemd reload? And why do we need a reload itself if systemd will be reloaded by the task above (if needed).

Or you're saying that `notify` for the task above won't be triggered right after that?

Reply:
Correct, what we specify in `notify` will be triggered at the very end of the playbook by default. We need to call `flush_handlers` to trigger them early in order to cover the edge case of the first run when units/sockets do not exist yet. Otherwise, the step below will fail.

Reply:
Gosh.. I always believed that `notify` is executed right after the task.

Reply:
I know, right? :)